### DISCUSSION TITLE
Alpine: EONOTSUPP or EOFOMISSION?

### DISCUSSION THREAD

**Discussion by skepticoitusInteruptus** [containers/podman#14418] [General]
Is it just me, or does anybody else find it surprising that [_Alpine is not listed as a supported distribution in Podman's docs_](https://bit.ly/EONOTSUPP):grey_question: 

From my experience (_and from what tons of blogs always say_) Alpine is probably the GOTO distro for Docker containers.

Seems like Podman would be all over it :confused:

**Comment by vrothberg** [comment]
Thanks for reaching out!

Note that the listed doc refers to the _host_ system, not the container image.  But Alpine (host) ships Podman now (see https://wiki.alpinelinux.org/wiki/Podman), so it may be worth updating the website to include that.

**Reply by vrothberg** [reply]
I opened https://github.com/containers/podman.io/pull/534 to add instructions to the website.

**Reply by skepticoitusInteruptus** [reply]
You rock, @vrothberg :rock: 

I see CI tests like…

* [_`Unit tests on fedora…`_](https://bit.ly/nun4Alpy)
* _`Windows Cross`_
* _`Unit tests on ubuntu…`_
* _`int podman fedora…`_
* _`sys podman ubuntu…`_
* _`OSX cross`_
* _`…`_

How soon after the docs website gets updated might analogous Alpine tests get added to Podman's CI process?

Guestimate-wise, of course.

**Reply by Luap99** [reply]
We cannot test every distro in the upstream CI. Currently only fedora and ubuntu is tested and I doubt that this will change anytime soon.

Testing for particular distro/package versions must be done by the packager and not upstream developers.

**Reply by skepticoitusInteruptus** [reply]
Hey there @Luap99 :wave:

> "_&hellip;Testing for particular distro/package versions must be done by the packager and not upstream developers&hellip;_"

Got it. Thanks.

So then what is the Podman maintainers' official(_-ish_) position on reports of/requests for Alpine-related bugs/features?

It's kind of hard (_for me_) to guess when or if a given Alpine-related issues is _leaving the supported territory_ :grey_question:

**Reply by Luap99** [reply]
We try to support as much as possible, generally speaking you should at least use a recent podman version (v4.0/v4.1 at the moment).
Almost all bugs/features will be applicable to all distros so there is no problem with those.
If there is really an alpine specific podman problem you can still create an issue but it is less likely that we can help. We always accept PRs if somebody manages to fix it.

**Reply by vrothberg** [reply]
I echo what @Luap99 wrote above.

Please open issues and bugs if you find some. While most maintainers work for Red Hat, we support all distributions equally upstream. As @Luap99 mentioned, we cannot test all distributions in the CI and we cannot provide packages for all distributions, but we are more than happy to collaborate with distributions.
